erpnext/crm/doctype/lead/lead.py

```diff
@@ -80,6 +80,22 @@ def has_customer(self):
 	def has_opportunity(self):
 		return frappe.db.get_value("Opportunity", {"lead": self.name, "status": ["!=", "Lost"]})
 
+	def has_quotation(self):
+		return frappe.db.get_value(
+			"Quotation", {
+				"lead": self.name,
+				"status": ["!=", "Lost"],
+				"status": ["!=", "Cancelled"]
```

Comment:
you cannot set same key twice in a dictionary, first one will be ignored. You can write like this `"status": ["not in", ["Lost", "Cancelled"]]`